codechecker_lib/analyzers/analyzer_types.py

```diff
@@ -189,7 +198,10 @@ def __build_clangsa_config_handler(args, context):
     config_handler.includes = context.extra_includes
     try:
         with open(args.clangsa_args_cfg_file, 'rb') as sa_cfg:
-            config_handler.analyzer_extra_arguments = sa_cfg.read().strip()
+            config_handler.analyzer_extra_arguments = \
+                re.sub('\$\((.*?)\)',
```

Comment:
This controls laziness: `abc$(def)ghi$(jkl)mno` would capture `def)ghi$(jkl` instead of `def` and `jkl`.